src/commands/codepush/release-react.ts

```diff
@@ -123,6 +123,10 @@ export default class CodePushReleaseReactCommand extends CodePushReleaseCommandB
   @hasArg
   public extraHermesFlags: string | string[];
 
+  @help("Enable hermes and bypass automatic checks")
+  @longName("use-hermes")
+  public useHermes: string | string[];
```

Comment:
Why it is not a `boolean`?

Reply:
Hi @DmitriyKirakosyan, thanks for the review!
You're right it should be a `boolean`